cms/static/coffee/spec/main_squire.coffee

```diff
@@ -176,6 +176,9 @@ requirejs.config({
         },
         "coffee/src/ajax_prefix": {
             deps: ["jquery"]
+        },
+        "draggabilly": {
```

Comment:
Probably don't actually need this here, though if the squire and non-squire versions are meant to be in sync, it's fine to leave it.